src/client.ts

```diff
@@ -1270,15 +1270,8 @@ export class Client {
       // Finally, serialize the transaction
       const serializedTx = bitcoin.serializeTx(preSerializedData);
       // Generate the transaction hash so the user can look this transaction up later
-      let preImageTxHash = serializedTx;
-      if (preSerializedData.isSegwitSpend === true) {
```

Comment:
I went ahead and removed this block, since it was impossible to be executed.

Reply:
Good catch